packages/listbox/__tests__/listbox.test.tsx

```diff
@@ -218,7 +222,12 @@ describe("<Listbox />", () => {
             </Listbox>
           </div>
         );
-        expect(container.querySelector("input")).toBeTruthy();
+        // Be careful not to match the `input[type=text]` added in this test
+        expect(container.querySelector("input[type=hidden]")).toBeTruthy();
+        expect(container.querySelector("input[type=hidden]")).toHaveAttribute(
```

Comment:
Let's remove these assertions, see comment above about future implementation changes.

Reply:
Sure. Do you mean restoring the test to how it was? Or just removing all assertions in this test?

Reply:
Yeah, we can revert these lines to the one line from before. Let's just keep the check for a standard input and use the `not.toBeVisible` method to make sure it's hidden.

Reply:
There are 2 inputs, the first added by the test (`<input type="text" name="name" />` not hidden) and the other added by `listbox` (which is hidden).

So the test you propose fails.

Do you like this option which will work to get that second input?
```
 expect(container.querySelectorAll("input")[1]).not.toBeVisible();

```

Reply:
I went ahead and pushed that up along with fixing the other line.